packager/packager.cc

```diff
@@ -1411,17 +1476,64 @@ ast::ParsedFile rewritePackage(core::Context ctx, ast::ParsedFile file) {
     return file;
 }
 
+string_view levelToSigil(core::StrictLevel level) {
```

Comment:
This is duplicated with the function in realmain.cc right? Is it time to factor that into `core::StrictLevel`?

Reply:
This has been factored out into the same place as `StrictLevel`. (Sadly, C++ doesn't let you add methods—static or instance—to `enum class` values, so it's a standalone method defined there instead.)